wordcloud/wordcloud.py

```diff
@@ -503,16 +508,17 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        regexp = self.regexp if self.regexp is not None else r"\w[\w']+"
+        regexp = self.regexp if self.regexp is not None else r"(\w[\w']+|\w+)"
```

Comment:
So you propose to make an additional regex for digits in case of having `render_numbers=True` or what is the idea? Because  the regex that I proposed accept numbers (`\w+` matches any word character, and it's equal to `[a-zA-Z0-9_]`) but it also needs an additional step to remove 1 char words. Please explain me more about your thoughts in order to know which should the best way to achieve this `render_numbers` feature.